xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/service/BittrexTradeService.java

```diff
@@ -106,10 +123,25 @@ public OpenOrdersParams createOpenOrdersParams() {
     try {
       List<Order> orders = new ArrayList<>();
       for (String orderId : orderIds) {
-        BittrexOrder order = getBittrexOrder(orderId);
-        if (order != null) {
-          LimitOrder limitOrder = BittrexAdapters.adaptOrder(order);
-          orders.add(limitOrder);
+        BittrexOrder bittrexOrder = getBittrexOrder(orderId);
+        if (bittrexOrder != null) {
+          Order order = BittrexAdapters.adaptOrder(bittrexOrder);
+          if (order instanceof MarketOrder) {
+            BigDecimal rate =
+                decorateApiCall(
+                        () -> {
+                          List<BittrexExecution> executions =
+                              getBittrexOrderExecutions(order.getId());
+                          if (executions != null && executions.size() > 0) {
+                            return executions.get(0).getRate();
+                          }
+                          throw new BittrexException();
```

Comment:
do you want to trigger this exception and cause this to use the 'withRetry' this way?